src/lib.rs

```diff
@@ -2,6 +2,12 @@
 //!
 //! # Cargo Features
 //!
+//! ## `sim`
+//!
+//! Enables special-case handling of the unique room name present in the
+//! simulator - must be enabled to build code that is compatible with that
+//! environment
```

Comment:
Again, a note on what exactly the behavior changes are would be nice.

Maybe the relevant method docs should also mention "if the `sim` feature is enabled..." too?